src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -287,6 +187,73 @@
       },
       "required": [ "value", "algorithm" ]
     },
+    "invocation": {
+      "description": "The runtime environment of the analysis tool run.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "parameters": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
+          "type": "string"
+        },
+
+        "startTime": {
+          "description": "The date and time at which the run started. See \"Date/time properties\" in the SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "endTime": {
+          "description": "The date and time at which the run ended. See \"Date/time properties\" in the  SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "correlationId": {
+          "description": "An identifier that allows the run to be correlated with other artifacts produced by a larger automation process.",
+          "type": "string"
+        },
+
+        "machine": {
```

Comment:
machine is expressed in an environment variable, if that dictionary is present, on windows machines. does this warrant its own top level property? what about current user account name (also expressed in env vars)?